src/loaders/libcuda.fn

```diff
@@ -30,6 +30,7 @@ DEF_PROC_V2(cuMemGetInfo, (size_t *free, size_t *total));
 DEF_PROC_V2(cuMemAlloc, (CUdeviceptr *dptr, size_t bytesize));
 DEF_PROC_V2(cuMemFree, (CUdeviceptr dptr));
 DEF_PROC_V2(cuMemAllocHost, (void **pp, size_t bytesize));
+DEF_PROC(cuMemAllocManaged, (CUdeviceptr* dptr, size_t bytesize, unsigned int flags));
```

Comment:
I tried to use DEF_PROC_V2 here and it failed. Why? How this work? I followed the code but didn't understood.